superset-frontend/src/pages/SavedQueryList/SavedQueryList.test.jsx

```diff
@@ -302,9 +302,13 @@ describe('RTL', () => {
     userEvent.click(bulkSelectButton);
 
     // Grab and click the "toggle all" checkbox to expose export button
-    const selectAllCheckbox = screen.getByRole('checkbox', {
-      name: /toggle all rows selected/i,
-    });
+    const selectAllCheckbox = screen
+      .getAllByRole('checkbox', {
+        name: '',
+      })
+      .filter(
+        checkbox => checkbox.getAttribute('name') === 'header-toggle-all',
+      )[0];
```

Comment:
The intention of this code seems clearer if the .find function is used.
```suggestion
      .find(
        checkbox => checkbox.getAttribute('name') === 'header-toggle-all',
      );
```